SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Security Policy
+
+## Supported Versions
+
+| Version   | Supported          |
+| -------   | ------------------ |
+|  0.3.0    | :white_check_mark: |
+| < 0.3.0   | :x:                |
+
+## Reporting a Vulnerability
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them via [email](mailto:info@auto-dl.tech).
```

Comment:
That works, but if someone doesn't have an outlook account they can't view the email.
So ig also just type out the email address, in case

Reply:
What? no no, it will open in any default email provider (opens in Gmail in my case)

Reply:
Try pasting `mailto:info@auto-dl.tech` in your browser

Reply:
That opens the website

Reply:
So it works right? 🤔

Reply:
Yes it works! xD